Author: Roland Mas <lolando@debian.org>
Description: Use Snoopy as provided by the libphp-snoopy Debian package
instead of shipping a local one.
Index: src/plugins/cvstracker/bin/post.php
===================================================================
--- src.orig/plugins/cvstracker/bin/post.php
+++ src/plugins/cvstracker/bin/post.php
@@ -32,7 +32,7 @@
 require dirname(__FILE__).'/../../env.inc.php';
 require_once $gfcommon.'include/pre.php';
 require_once ($gfcommon.'common/include/utils.php');
-require_once ($gfplugins.'cvstracker/common/Snoopy.class.php');
+require_once ('/usr/share/php/libphp-snoopy/Snoopy.class.php');
 
 $cvs_binary_version = get_cvs_binary_version () ;
 
Index: src/plugins/scmcvs/bin/aclcheck.php
===================================================================
--- src.orig/plugins/scmcvs/bin/aclcheck.php
+++ src/plugins/scmcvs/bin/aclcheck.php
@@ -31,7 +31,7 @@
 require_once $gfcommon. 'include/pre.php';
 require_once $gfcommon.'include/utils.php';
 require_once $gfconfig.'plugins/scmcvs/config.php';
-require_once $gfplugins.'scmcvs/common/Snoopy.class.php';
+require_once '/usr/share/php/libphp-snoopy/Snoopy.class.php';
 
 // Input cleansing
 $env_cvsroot = (string) $_ENV['CVSROOT'];
